anomalib/data/__init__.py

```diff
@@ -132,6 +133,24 @@ def get_datamodule(config: Union[DictConfig, ListConfig]) -> AnomalibDataModule:
             val_split_mode=config.dataset.val_split_mode,
             val_split_ratio=config.dataset.val_split_ratio,
         )
+    elif config.dataset.format.lower() == "visa":
+        datamodule = Visa(
+            root=config.dataset.path,
```

Comment:
I prefer `root` because this should point to the root directory of the dataset, without category information. When using `path` it would be less clear if the user should provide the path to the root (e.g. `datasets/mvtec/`) or the category (`datasets/mvtec/cable`). We could update the description to `root (Path): Path to the root directory of the dataset`.

If you disagree I am fine with using `path` as well.

Reply:
i'm neutral, but just curriout to know which one would be the best. @ashwinvaidya17, what is your thought here?

Reply:
I agree that `root` is more clear. And in the new configs we also have `data.init_args.root` that points to the dataset root. Personally I don't mind if we use both `root` and `path` in the repo to make the distinction clear. We can use `path` in the cases where we point to a particular folder that contains images.

Reply:
Alright, thanks for your inputs!